modules/functional-test/src/main/resources/config.properties

```diff
@@ -39,6 +39,7 @@ api-server-protocol=http
 api-version-string=v1
 user-name=admin
 password=admin
+test-user=wasabi_reader@example.com
```

Comment:
sure @jcwuzoegiver i will incorporate the change. thanks for the very good suggestion.